src/app/components/Markdown.tsx

```diff
@@ -1,84 +1,96 @@
+const limitedMarkdownPlugin: Plugin = () => {
+    return (tree, file) => {
+        const contents = file.toString();
 
-                    return <Blockquote {...rest}>{children}</Blockquote>;
-                },
+        visit(tree, (node, index, parent) => {
+            if (
+                index == null ||
+                [
+                    'paragraph',
+                    'text',
+                    'inlineCode',
+                    'strong',
+                    'emphasis',
+                    'link',
+                    'list',
+                    'listItem',
+                    'blockquote',
+                ].includes(node.type) ||
+                !node.position
+            ) {
+                return true;
+            }
 
-                code(props) {
-                    const { children, className, node, ...rest } = props;
-                    const match = /language-(\w+)/.exec(className || '');
+            let value = contents.slice(node.position.start.offset, node.position.end.offset);
 
-                    return match ? (
-                        <code {...rest} className={className}>
-                            {children}
-                        </code>
-                    ) : (
-                        <code {...rest} className={classNames('bg-slate-200 rounded', className)}>
-                            {children}
-                        </code>
-                    );
-                    // return match ? (
-                    //   <SyntaxHighlighter
-                    // 	{...rest}
-                    // 	PreTag="div"
-                    // 	children={String(children).replace(/\n$/, '')}
-                    // 	language={match[1]}
-                    // 	style={dark}
-                    //   />
-                    // ) : (
-                    //   <code {...rest} className={className}>
-                    // 	{children}
-                    //   </code>
-                    // )
-                },
+            if (node.type === 'heading') {
+                value = `\n${value}`;
+            }
 
-                h1(props) {
-                    const { node, children, ...rest } = props;
+            parent.children[index] = {
+                type: 'paragraph',
+                children: [
+                    {
+                        type: 'text',
+                        value,
+                    },
+                ],
+            } as any;
 
-                    return <H1 {...rest}>{children}</H1>;
-                },
-                h2(props) {
-                    const { node, children, ...rest } = props;
+            return [SKIP, index] as const;
+        });
+    };
+};
 
-                    return <H2 {...rest}>{children}</H2>;
+export default function Markdown(props: Options & { extraClassName: string }) {
+    return (
+        <ReactMarkdown
+            className={classNames('text-lg text-balance hyphens-auto break-words [&>*]:mb-3', props.extraClassName)}
```

Comment:
```suggestion
            className={classNames('text-lg whitespace-pre-wrap text-balance hyphens-auto break-words [&>*]:mb-3', props.extraClassName)}
```

Reply:
Uhmm, I just checked and GitHub behaves the same way. Might be a good use case for the `pre` element... (`` ``` `` in markdown). I'll enable it.

Reply:
Aah yes, good point. That would solve it (at least for future comments).